bundle/manifests/compliance-operator.clusterserviceversion.yaml

```diff
@@ -1600,8 +1610,8 @@ spec:
   - openscap
   - audit
   links:
-  - name: Compliance Operator upstream repository
-    url: https://github.com/ComplianceAsCode/compliance-operator
+  - name: Compliance Operator upstream repo
+    url: https://github.com/openshift/compliance-operator
```

Comment:
For some reason this is getting overwritten even though we fixed it - https://github.com/ComplianceAsCode/compliance-operator/pull/230/files

Reply:
This should be fixed when we rebase on https://github.com/ComplianceAsCode/compliance-operator/pull/275